app/views/help/refusal_advice/_action.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="wizard__action" data-block="<%= action.id %>">
   <h2><%= action.header %></h2>
 
+  <%= render action.body %>
```

Comment:
Wrap in a `<div>` so that we're sure it's enclosed in a block element to prevent the button appearing on the same line:

![Screenshot 2021-02-09 at 12 15 21](https://user-images.githubusercontent.com/282788/107362708-06386380-6ad1-11eb-8d0f-32f9e2c61965.png)

Reply:
Might as well also wrap `<%= render action.suggestions %>` in a `<div>` too.

Reply:
Don't think `<%= render action.suggestions %>` needs this as the partial already wraps